pyanaconda/modules/payloads/base/utils.py

```diff
@@ -35,19 +35,19 @@ def create_root_dir(sysroot):
     mkdirChain(os.path.join(sysroot, "root"))
 
 
-def write_module_blacklist(sysroot):
-    """Create module blacklist based on the user preference.
+def write_module_blocklist(sysroot):
+    """Create module blocklist based on the user preference.
 
     Copy modules from modprobe.blacklist=<module> on cmdline to
     /etc/modprobe.d/anaconda-blacklist.conf so that modules will
-    continue to be blacklisted when the system boots.
+    continue to be blocklisted when the system boots.
     """
     if "modprobe.blacklist" not in kernel_arguments:
         return
 
     mkdirChain(os.path.join(sysroot, "etc/modprobe.d"))
     with open(os.path.join(sysroot, "etc/modprobe.d/anaconda-blacklist.conf"), "w") as f:
```

Comment:
Don't we want to change the name of the file as well, or is it documented somewhere?

Reply:
Never mind. I have found https://github.com/rhinstaller/anaconda/pull/3203.